modules/calendar.md

```diff
@@ -28,7 +28,7 @@ The following properties can be configured:
 | `maximumEntries`             | The maximum number of events shown. / **Possible values:** `0` - `100` <br> **Default value:** `10`
 | `maximumNumberOfDays`        | The maximum number of days in the future. <br><br> **Default value:** `365`
 | `displaySymbol`              | Display a symbol in front of an entry. <br><br> **Possible values:** `true` or `false` <br> **Default value:** `true`
-| `defaultSymbol`              | The default symbol. <br><br> **Possible values:** See [Font Awesome](https://fontawesome.io/icons/) website. <br> **Default value:** `calendar`
+| `defaultSymbol`              | The default symbol. <br><br> **Possible values:** See [Font Awesome](https://fontawesome.com/v4.7/icons/) website. <br> **Default value:** `calendar`
```

Comment:
thanks fpr catching this. but maybe point it to the more general url: https://fontawesome.com/icons/ ?
we are already using v5 IIRC

Reply:
It looks like a lot of icon on that are not supported. I suppose only the `free` icons are supported, but even there there are different styles: regular, bold, light... any idea what the right filter might be?

Reply:
we are using v5.15.4 and the url mentioned by @rejas is redirected to v15. To get only the free icons you can set a filter so may this would be the correct solution: https://fontawesome.com/icons?m=free

Reply:
But what about the style? If I would like to use this icon, I cannot do it because it requires a different class (`fas` and noch `fa`). 
![763FF728-982E-47E7-8A1D-0144338CBE22](https://user-images.githubusercontent.com/9592452/148172157-ad3df3df-d783-4723-af16-881d9eae9cb5.jpeg)

Reply:
Just [found](https://fontawesome.com/v5.15/how-to-use/on-the-web/referencing-icons/basic-use) this:

> The fa prefix has been deprecated in Version 5. The new default is the fas solid style and the fab style for brands.

Reply:
So we should rename the icons used?